apps/etterlatte-vedtaksvurdering/src/main/kotlin/vedtaksvurdering/grunnlag/GrunnlagVersjonValidering.kt

```diff
@@ -12,9 +13,13 @@ object GrunnlagVersjonValidering {
     fun validerVersjon(
         vilkaarsvurdering: VilkaarsvurderingDto?,
         beregningOgAvkorting: BeregningOgAvkorting?,
+        trygdetid: TrygdetidDto?,
     ) {
         logger.info("Sjekker at grunnlagsversjon er konsekvent på tvers av appene")
```

Comment:
Kunne eventuelt hatt en funksjonsbryter for om denne sjekken skal kjøres? Så er man bedre rustet opp mot endringer i produksjon som kan blokkere arbeid hvis det er noen bugs. Gjør det mindre risikofylt å merge inn og få testet det i dev f.eks. :)